### PR TITLE
added support for mongoid 6 beta release

### PR DESCRIPTION
**Pull request by narutosanjiv**
In mongoid 6, origin seem to be removed and they replaced it various modules

### CODE REVIEW

**Comment by narutosanjiv** [comment]
@rodrigopinto if possible, have a look at PR.

**Comment by matheusvetor** [comment]
👍

File: lib/matchers/associations.rb [hunk 1]
```diff
@@ -38,8 +37,8 @@ def ordered_by(association_field_name)
           raise "#{@association[:type].inspect} does not respond to :order" unless [HAS_MANY, HAS_AND_BELONGS_TO_MANY, EMBEDS_MANY].include?(@association[:type])
           @association[:order] = association_field_name.to_s
           @expectation_message << " ordered by #{@association[:order].inspect}"
-
-          if association_field_name.is_a? Origin::Key
+          query_key = (Mongoid::VERSION =~ /^6/).blank? ? Origin::Key : Mongoid::Criteria::Queryable::Key
+          if association_field_name.is_a? query_key
```
Comment (rodrigopinto) [inline lib/matchers/associations.rb]: Thanks for you contribution. 👏 

Could we just improve it a little to make it more readable? Maybe move the condition to a specific a predicate method and call it on another method that returns the correct `key`. What do you think about it?

**Comment by narutosanjiv** [comment]
@rodrigopinto Yeah, it seem fantastic idea, will do it.

File: gemfiles/Gemfile.mongoid-6.0 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'mongoid', '~> 6.0.0.beta'
```
Comment (gurix) [inline gemfiles/Gemfile.mongoid-6.0]: 6.0.0.rc0 is out so you could also update this

**Comment by gurix** [comment]
Probably you should also update the Readme.md

**Comment by narutosanjiv** [comment]
@gurix  OK , will update

**Comment by narutosanjiv** [comment]
@gurix  Update to mongoid-6.0.rc0 as well as Readme

**Comment by dbackeus** [comment]
👍

**Comment by 90yukke** [comment]
Closing this PR since beta support of mongoid 6 is already in master. See #179 for details.

**Comment by dbackeus** [comment]
@90yukke nice, would appreciate a new gem release as well 👍

**Pull request closed without merging**